apps.js

```diff
@@ -0,0 +1,118 @@
+class Task {
+	constructor(num, string) {
+    this.num = num;
+    this.string = string;
+	 }
+  }
+
+	class IntBuilder extends Task {
+	constructor(num) {
+    super(num);
+  
+     }  
+     plus() {
+        for (let i = 0; i < arguments.length; i++) {
+        this.num += arguments[i];
+        }
+        this.num;
+        return this;
+       }
+       
+     minus() {
+         for (let i = 0; i < arguments.length; i++) {
+             this.num -= arguments[i];
+         }
+         this.num;
+         return this;
+     }
+     
+     multiply(n) {
+        this.num *= n;
+        return this;
+    
+    }
+        divide(n) {
+        this.num /= n;
+        return this;
+    }
+
+        mod(n) {
+        this.num %= n;
+        return this;
+    }
+
+    static random(x, y) {
+        Math.floor(Math.random() * (y - x) + x);
```

Comment:
you shouldn't return "this" in random method
It's static, just return counted value